tests/components/application-connector/README.md

```diff
@@ -0,0 +1,2 @@
+# Application Gateway tests
```

Comment:
It's not only the Application Gateway test. It's a whole Application Connector test. At least in theory for now

Reply:
I will write here some docs how to run it